### PR TITLE
Added possibility of writing LAS v1.0 files when using PyHelios

### PR DESCRIPTION
**Pull request by miguelyermo-heidelberg**
The behaviour is exactly the same as the implemented in the main program. Nevertheless, I may have spotted an incongruency in the trajectory output formats:

image

Could anyone confirm that these are the expected trajectory formats? I expected to have .bin formats only when the flag 
```--zipOutput``` is passed to the execution.

### CODE REVIEW

**Comment by lwiniwar** [comment]
thank you for the comment - indeed, this is not as expected.
In my opinion, compressing trajectory files is not needed, as they contain much less data than point clouds.
I would therefore expect (wish for) the following:

| Flag                            | PC format  | Trajectory format |
|---------------------------------|------------|-------------------|
| (none)                          | .txt       | .txt              |
| --lasOutput                     | .las (1.4) | .txt              |
| --lasOutput --las10             | .las (1.0) | .txt              |
| --las10                         | .las (1.0) | .txt              |
| --zipOutput                     | .bin       | .txt              |
| --zipOutput --lasOutput         | .laz (1.4) | .txt              |
| --zipOutput --lasOutput --las10 | .laz (1.0) | .txt              |
| --zipOutput --las10             | .laz (1.0) | .txt              |

**Comment by han16nah** [comment]
I agree, I would also prefer having the .las file compressed but not the trajectory.

File: pyhelios_demo/pyhelios_argparser.py [hunk 1]
```diff
@@ -38,6 +38,9 @@
 parser.add_argument('--lasOutput', dest='las_output_flag', action='store_const', const=True, default=False,
                     help='Use this flag to generate the output point cloud in LAS format.')
 
+parser.add_argument('--las10', dest='las10_output_flag', action='store_const', const=True, default=False,
+                    help='Use this flag along with --lasOutput to write LAS v1.0 files.')
```
Comment (lwiniwar) [inline pyhelios_demo/pyhelios_argparser.py:42]: using the flag on its own should be reflected in the pyhelios argparser

File: src/LidarSim.h [hunk 2]
```diff
@@ -30,6 +30,7 @@ class LidarSim {
      *  false means previously built scene will be used when available
      * @param lasOutput Flag to specify LAS output format. True implies using
      *  LAS output format, false implies don't
+     * @param las10 Flag to specify that the output format mmust be LAS v1.0.
```
Comment (lwiniwar) [inline src/LidarSim.h:33]: typo in "must"

File: src/surveyplayback/SurveyPlayback.h [hunk 3]
```diff
@@ -26,6 +26,11 @@ class SurveyPlayback : public Simulation {
 	 *  or not (false)
 	 */
 	bool lasOutput = false;
+        /**
+         * @brief Flag to specify if LAS output must be LAS v1.0. Used along
+         * with --lasOutput
```
Comment (lwiniwar) [inline src/surveyplayback/SurveyPlayback.h:31]: not necessarily used together with --lasOutput

**Review by lwiniwar — COMMENTED** [review]
looks good, some minor comments below.

**Review by lwiniwar — APPROVED** [review]
lgtm!

**Pull request merged**